components/molecules/HighlightInput/highlight-input-form.tsx

```diff
@@ -61,6 +63,10 @@ const HighlightInputForm = (): JSX.Element => {
   const handlePostHighlight = async (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault();
 
+    if (wordCount > wordLimit) {
+      ToastTrigger({ message: "Character limit exceeded", type: "error" });
```

Comment:
I'd rather disable the save button instead of showing an alert. We already indicate their over the limit in the textarea